### ISSUE TITLE
K8s NodePool labels and annotations are missing.

### ISSUE THREAD

**Issue by dreamblack86** [ionos-cloud/terraform-provider-ionoscloud#31]
## Description
We need the ability to mark our K8s nodes with labels. This is already possible via the IONOS Cloud API v6, but this Terraform provider has not implemented this in itself. Will this be implemented in the future?

## Expected behavior
Details of the Kubernetes Node Pool via IONOS-Cloud API
Example Value
Model
```json
{
  "metadata": {},
  "properties": {
    "name": "k8s-node-pool",
    "labels": {
      "additionalProp1": "string",
      "additionalProp2": "string",
      "additionalProp3": "string"
    },
    "annotations": {
      "additionalProp1": "string",
      "additionalProp2": "string",
      "additionalProp3": "string"
    }
}
```

## Environment

Terraform version: 
```
0.14.3
```
Provider version: 
```
6.0.0-alpha.2
```
OS:
```
NAME="Ubuntu"
VERSION="20.04.2 LTS (Focal Fossa)"
```

## References
[https://editor.swagger.io/?url=https://api.ionos.com/cloudapi/v6/swagger.json](https://editor.swagger.io/?url=https://api.ionos.com/cloudapi/v6/swagger.json)

**Comment by dreamblack86** [comment]
Hi @LiviusP, Is there a reason why it is not supported in Terraform?

**Comment by mflorin** [comment]
Hello @dreamblack86 , we're currently depending on an API issue to be solved in order to roll out our terraform implementation. I'll keep you posted when the provider will become available.

**Comment by mflorin** [comment]
Hello @dreamblack86  - k8s node pool labels and annotations are now available starting with `v6.0.0-beta.9`.
